common/clean-chroot-manager64.in

```diff
@@ -221,9 +226,15 @@ build() {
 	local mesg="Attempting to build package..."
 	echo -e "${YELLOW}---->${ALL_OFF}${BOLD} ${mesg}${ALL_OFF}"
 
+        
         [[ -n "$RUNNAMCAP" ]] && RUNNAMCAP=-n
+        [[ -n "$USE_CCACHE" ]] && { USE_CCACHE=-d$CCACHE_DIR
+                                    local ccache_dir="CCACHE_DIR=$CCACHE_DIR" }
         
-	PACKAGER=$PACKAGER MAKEFLAGS=-j$THREADS nice -19 /usr/bin/makechrootpkg -c -u $RUNNAMCAP -r "$CHROOTPATH64"
+  	PACKAGER=$PACKAGER MAKEFLAGS=-j$THREADS nice -19 /usr/bin/makechrootpkg $USE_CCACHE \
+                -c -u $RUNNAMCAP \
+                -r "$CHROOTPATH64" \
+                -- $ccache_dir
 	
```

Comment:
Shouldn't you just ask for a CCACHE_DIR and then bind-mount it directly as:
 `-d "$CCACHE_DIR:/build/.ccache"`?

Keeping in mind also that, regardless, user-supplied paths can contain spaces which will break your PR.
I suggest assembling the extra arguments via a (properly-quoted) array.
You should be able to unify build() and buildnc() too.